### PR TITLE
refactor(fingerprint): replace `@expo/babel-preset-cli` with `expo-module-scripts`

### PR DESCRIPTION
**Pull request by byCedric**
~~⚠️ On hold until https://github.com/expo/expo/pull/25458 is merged.~~ → ✅ merged & (re)based

# Why

We are deprecating the `@expo/babel-preset-cli`, and that means that other packages need to move over to `expo-module-scripts`.

See expo/expo#25426 for all PRs.

# How

- Added `expo-module-scripts@^3.3.0` to `devDependencies`
- Replaced **package.json** scripts with `expo-module-scripts` equivalents

# Test Plan

See if CI passes.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/@expo/fingerprint/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Replace `@expo/babel-preset-cli` with `expo-module-scripts`.
```
[the remark this answers is not in the document]
Comment (byCedric) [reply packages/@expo/fingerprint/CHANGELOG.md:13]: docs(fingerprint): fix changelog entry

File: packages/@expo/fingerprint/package.json [hunk 2]
```diff
@@ -5,17 +5,21 @@
...
+    "test": "expo-module test",
+    "test:e2e": "expo-module prepare && expo-module test --config e2e/jest.config.js",
+    "typecheck": "expo-module typecheck"
   },
   "bin": {
     "fingerprint": "bin/cli.js"
   },
+  "files": [
+    "build"
+  ],
```
Comment (byCedric) [inline packages/@expo/fingerprint/package.json:22]: @Kudo The `files` property was missing from this `package.json`. I believe this should be pretty much correct, but if `scripts` also need to be included in the published package, lmk!

File: packages/@expo/fingerprint/package.json [hunk 3]
```diff
@@ -5,17 +5,21 @@
...
+    "prepublishOnly": "expo-module prepublishOnly",
+    "test": "expo-module test",
+    "test:e2e": "expo-module prepare && expo-module test --config e2e/jest.config.js",
+    "typecheck": "expo-module typecheck"
   },
   "bin": {
     "fingerprint": "bin/cli.js"
   },
+  "files": [
+    "build"
```
Comment (Kudo) [inline packages/@expo/fingerprint/package.json:21]:
```suggestion
    "bin",
    "build"
```

we also need the `bin/`

**Review by Kudo — APPROVED** [review]
thanks Cedric for the help.

**Review by Kudo — APPROVED** [review]

**Pull request merged**